ddtrace/tracer/tracer.go

```diff
@@ -143,24 +170,51 @@ func newTracer(opts ...StartOption) *tracer {
 		prioritySampling: newPrioritySampler(),
 		pid:              strconv.Itoa(os.Getpid()),
 	}
+	t.config.statsd.Incr("datadog.tracer.started", nil, 1)
 	if c.runtimeMetrics {
-		statsd, err := statsd.NewBuffered(t.config.dogstatsdAddr, 40)
-		if err != nil {
-			log.Warn("Runtime metrics disabled: %v", err)
-		} else {
-			log.Debug("Runtime metrics enabled.")
-			go t.reportMetrics(statsd, defaultMetricsReportInterval)
-		}
+		log.Debug("Runtime metrics enabled.")
+		go t.reportMetrics(defaultMetricsReportInterval)
```

Comment:
Should this be attached to the waitgroup or is it intentionally independent, ie: needs to end after worker/statsSender?

Reply:
No, this should be part of the wait group as well.